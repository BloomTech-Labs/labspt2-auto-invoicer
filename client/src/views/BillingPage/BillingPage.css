.billing-container {
  align-items: center;
  display: flex;
  flex-direction: column;
  width: 100%;
  min-height: 450px;
  justify-content: space-between;
  padding-top: 20px;
}
