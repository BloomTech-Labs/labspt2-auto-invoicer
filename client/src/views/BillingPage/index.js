import React from 'react';

// import components here
import StripeElements from './../../components/StripeElements';

// import styles here
import './BillingPage.css';

const BillingPage = () => {
  return (
    <section className="billing-container">
      <StripeElements />
    </section>
  );
};

export default BillingPage;
