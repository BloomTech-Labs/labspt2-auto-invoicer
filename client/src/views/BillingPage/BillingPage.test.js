import React from "react";
import { render } from "react-testing-library";
import "jest-dom/extend-expect";
import BillingPage from "../BillingPage";

describe("<BillingPage />", () => {
  it("renders Billing Page view", () => {
    const { asFragment } = render(<BillingPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
