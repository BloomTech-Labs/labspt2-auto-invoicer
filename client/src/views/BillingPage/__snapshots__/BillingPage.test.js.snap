// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BillingPage /> renders Billing Page view 1`] = `
<DocumentFragment>
  <section
    class="billing-container"
  >
    <h1
      class="main-title"
    >
      Billing
    </h1>
    <form
      class="billing_form"
    >
      <div
        class="payment_area"
      >
        <fieldset
          class="payment_info"
        >
          <legend>
            Payment Info
          </legend>
          <input
            maxlength="16"
            name="cc"
            pattern="[0-9]{16}"
            placeholder="Credit Card #"
            required=""
            type="text"
            value=""
          />
          <input
            maxlength="4"
            name="exp"
            pattern="[0-9]{4}"
            placeholder="Expiration Date"
            required=""
            type="text"
            value=""
          />
          <input
            maxlength="3"
            name="cvv"
            pattern="[0-9]{3}"
            placeholder="Cvv"
            required=""
            type="text"
            value=""
          />
        </fieldset>
        <button
          class="button"
          type="submit"
        >
          Buy Now
        </button>
      </div>
      <section>
        <label
          class="payment_checkboxes"
          for="9.99"
        >
          <input
            id="9.99"
            name="payment"
            type="radio"
            value="9.99"
          />
          <p>
            1 Month Subscription - $9.99
          </p>
        </label>
        <label
          class="payment_checkboxes"
          for="0.99"
        >
          <input
            id="0.99"
            name="payment"
            type="radio"
            value="0.99"
          />
          <p>
            1 Invoice - $0.99
          </p>
        </label>
      </section>
    </form>
  </section>
</DocumentFragment>
`;
